### PR TITLE
Balance check upload

### LINKED ISSUE #834: CREATE - Upload Account - Block + Inform user if file size exceeds upload account balance
- [x] block release create if not enough sol to create a release (event amount)
- [x] block release after file upload if upload acct cant cover file size

### PR DESCRIPTION
**Pull request by ikjudd**
- Closes #834

### CODE REVIEW

File: js/sdk/src/components/MediaDropzone.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ const MediaDropzone = ({
       return (
         <Box style={{ ...previewBoxStyles, ...audioPreviewStyles }}>
           {cancelIcon(fileWithMeta.remove)}
-          <Box sx={{ padding: '35px 15px' }}>
+          <Box sx={{ padding: '35px 15px' }} disabled={true}>
```
Comment (ejf89) [inline js/sdk/src/components/MediaDropzone.js:130]: ?

**Review by ejf89 — APPROVED** [review]

**Review by facing-n — CHANGES_REQUESTED** [review]
1. the LowSolWarning component shouldn't be a modal.  it should be over the ReleaseCreateForm.  You shouldn't be able to manually dismiss the warning - instead the LowSolWarning should periodically call getUserBalances (every 5 seconds?) and dismiss warning when you have the right balance.

2. the required SOL amount is incorrect.  there is a diff cost if through a Hub - to err on the safe side we probably want to round up from NinaProgramActionCost.RELEASE_INIT_VIA_HUB, which should display as 0.023

File: js/sdk/src/components/ReleaseCreate.js [hunk 2]
```diff
@@ -468,13 +497,19 @@ const ReleaseCreate = ({ canAddContent, hubPubkey }) => {
     }
   }
 
+  function roundUp(number, decimal) {
```
Comment (ejf89) [inline js/sdk/src/components/ReleaseCreate.js:500]: plz move this to `/Utils`

**Review by ejf89 — CHANGES_REQUESTED** [review]
plz move `roundUp` helper function to Utils

**Review by ejf89 — APPROVED** [review]

**Review by facing-n — APPROVED** [review]

**Pull request merged**